### PR TITLE
project export/import test

### PR DESCRIPTION
**Pull request by mwdchang**
### Summary
Testing steps for project export/import
- Simple workflow involving simulate
- Simple workflow involving data-transform

### CODE REVIEW

**Review by YohannParis — APPROVED** [review]
Awesome!

File: testing/manual/project-export-import.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    ```
+    email: qa@test.io
+    password: askem-quality-assurance
+    ```
+
+### 2. Scenario A: Simulate workflow
+1. Create a new project
+2. Add a simple model into the project, eg SIR
+3. Create a simple simulation workflow: model => model-config => simulate
+4. Run simulate, pick a few variables to plot once simulate compltes
```
Comment (mloppie) [inline testing/manual/project-export-import.md:18]: typo

File: testing/manual/project-export-import.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+### 3. Scenario B: Data transform workflow
+1. Create a new project
+2. Add a dataset into the project
+3. Create a simple data-transform workflow: dataset => data-transform
+4. Run a few cells in data-transform, eg: show rows/columns, plot a few charts
+5. Export the project: Project-overview, download
+6. Rename the project to [project-old]
+7. Import the project you downloaded
+8. Once import completes, go into the  new project, verify that you can open the data-transform and see previous cells and cell-results
```
Comment (mloppie) [inline testing/manual/project-export-import.md:35]: extra space

**Review by mloppie — APPROVED** [review]

File: testing/manual/project-export-import.md (hunk 1 above)
Comment (mwdchang) [reply testing/manual/project-export-import.md]: thx

**Pull request merged**